### DISCUSSION TITLE
student developer pack

### DISCUSSION THREAD

**Discussion by hruthikgundla** [community/community#50422] [GitHub Education]
I'm getting an error while applying for GitHub student developer pack like location error etc. can any one help me. please look at the screenshot attached below.
 Thanks in advance.
<img width="1440" alt="Screenshot 2023-03-19 at 12 00 54 PM" src="https://user-images.githubusercontent.com/96756682/226172195-3aadc415-fd60-40ab-b8d0-d92c197e9be1.png">

**Comment by mtflyfisher** [comment]
Did you ever get this figured out? I have the same issues minus the second reason. I am an online student and they even verified my college email yet still unable to get access. Frustrating.

**Comment by samus-aran** [comment]
Hi @hruthikgundla 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

For remote student applicants: please ensure that your evidence indicates that you are a remote student. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status.
